src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -282,7 +292,7 @@ public String getInvalidityInfoForEmail(String email) {
         String sanitizedValue = SanitizationHelper.sanitizeForHtml(email);
 
         if (email.isEmpty()) {
-            return getPopulatedErrorMessage(EMAIL_ERROR_MESSAGE, email, EMAIL_FIELD_NAME, REASON_EMPTY,
+            return getPopulatedErrorMessage(EMAIL_ERROR_MESSAGE_EMPTY_STRING, email, EMAIL_FIELD_NAME, REASON_EMPTY,
```

Comment:
@amarlearning I believe since that message has been created just for empty string errors, there is little reason why it is necessary for users of that string to fill up the reason manually or fill in redundant fields using existing methods. Of course, what I suggested may not be the best way to improve your code, but I do think there is room for improvement.